src/components/organisms/Cart/style.scss

```diff
@@ -0,0 +1,47 @@
+.header__cart {
+    display: flex;
+    border-right: 2px solid var(--main-color);
+    margin-right: 10px;
+}
+.header__count {
```

Comment:
My suggestion to put empty line between classes.

Reply:
changed